app/models/utils/nav/customization_parser.rb

```diff
@@ -5,10 +5,45 @@ module Nav
     # Parses the top menu customization setting
     class CustomizationParser
       attr_accessor :markdown
+      attr_accessor :customizations
 
       def initialize(markdown)
         # a nil input is stored as an empty string
         self.markdown = markdown || ""
+        self.customizations = parse
+        @done = []
+      end
+
+      def filter_item(name:, path:, permitted:, icon:)
+        # check to see if this item has been customized
+        result = {}
+        result[:name] = name
+        translated_name = I18n.t("nav_links.main.#{name}")
+        if @customizations[translated_name]
```

Comment:
No need for the `@` since you have the accessor. Skipping the `@` is considered best practice.

Reply:
okay.